docs/general/administration/hardware-selection.md

```diff
@@ -17,7 +17,7 @@ For a Jellyfin server, the following is recommended:
 - CPU (Without dGPU): Intel Pentium G4560, Intel Core i3-7100 or better. (Intel 7th gen or newer Pentium or better, excluding J and N series)
 - RAM: 8GB or more
 - Storage: 60GB SSD storage for Jellyfin files and transcoding cache.
-- Graphics: Intel HD 6xx (7th gen integrated graphics) or newer, Nvidia GTX 16 / RTX 20 series or newer (excluding GTX 1650). Intel is recommended over Nvidia. AMD and Apple Silicon are not recommended.
+- Graphics: Intel HD 6xx (7th gen integrated graphics) or newer, Nvidia GTX 16 / RTX 20 series or newer (excluding GTX 1650). Intel is recommended over Nvidia. AMD is not recommended.
```

Comment:
The reasons Apple Silicon wasn't recommended when this guide was written are as follows:
1. No jellyfin-ffmpeg support
2. Poor encoding quality compared to QSV, NVENC and even AMF during my testing.

Reply:
for 2; That is not the case anymore then I assume?

Reply:
The upstream ffmpeg had a very broken VideoToolbox implementation until very recently. I back-ported  some patches to our own ffmpeg to make at least the full-range color works. But to verdict this I still need to run benchmarks to give us numbers.

Reply:
Ran a quick test against an 30M bitrate 1080p movie, compressing to 3000K HEVC.

Here's the VMAF test result:

- NVENC: 97.845684
- QSV: 97.743323
- VideoToolbox: 97.579240

So the score is indeed lower, but not to an extent where "not recommend" should be labeled, at least on this sample and this bitrate. I will run additional tests to see if things are different.

Reply:
Yea that delta between VideoToolbox and QSV is not something a user should worry about. I agree with the removal of not recommending apple silicon then, if this is the quality users can expect.

Reply:
I run an additional sample where all encoders performs bad at 3000K HEVC:

- NVENC: 59.085323
- QSV: 58.456923
- VideoToolbox: 59.149807

And in this case VT is slightly better in score.  But it does not make any sense to human as the visual quality is just so bad for all of them.

Reply:
With the same sample not giving ideal score, but now use the 3000K H264:

- NVENC: 58.908556
- QSV: 58.832961
- VideoToolbox: 58.740689

This turn VideoToolbox back to the lowest, but still no meaningful delta.